class.jetpack.php

```diff
@@ -2690,7 +2690,9 @@ public static function activate_default_modules( $min_version = false, $max_vers
 			}
 
 			// we'll override this later if the plugin can be included without fatal error
-			wp_safe_redirect( Jetpack::admin_url( 'page=jetpack' ) );
+			if ( $redirect ) {
```

Comment:
According to the conditional above (line 2364), `$redirect` is always true at this point.  Should it be `||` instead of `&&` above?